admin/notifications/views.py

```diff
@@ -9,29 +10,32 @@ def delete_selected_notifications(selected_ids):
 
 def detect_duplicate_notifications():
     duplicates = (
-        NotificationSubscription.objects.values('user', 'node', 'event_name')
-        .annotate(count=Count('id'))
+        NotificationSubscription.objects.values('_id')
+        .annotate(count=Count('_id'))
         .filter(count__gt=1)
     )
 
     detailed_duplicates = []
     for dup in duplicates:
         notifications = NotificationSubscription.objects.filter(
-            user=dup['user'], node=dup['node'], event_name=dup['event_name']
+            _id=dup['_id']
         ).order_by('created')
 
-        for notification in notifications:
+        if notifications.exists():
+            notification = notifications.first()
             detailed_duplicates.append({
                 'id': notification.id,
-                'user': notification.user,
-                'node': notification.node,
+                '_id': notification._id,
                 'event_name': notification.event_name,
                 'created': notification.created,
-                'count': dup['count']
+                'count': dup['count'],
+                'email_transactional': [u._id for u in notification.email_transactional.all()],
+                'email_digest': [u._id for u in notification.email_digest.all()]
```

Comment:
`none` is the third relevant M2M here; it contains users who have unsubscribed
```suggestion
                'email_digest': [u._id for u in notification.email_digest.all()],
                'none': [u._id for u in notification.none.all()]
```